### ISSUE TITLE
add qiime2

### ISSUE THREAD

**Issue by colinbrislawn** [biocore/conda-recipes#61]
The future is arriving! And when it comes, it should be in a conda package. :snake: :package: 

I'll take this one, and make sure it stays up and running throughout development. @gregcaporaso already got it building in travis.

**Comment by gregcaporaso** [comment]
I agree, and thanks for volunteering. Hold off on this one for a bit now - getting this in place around mid-to-late March would be ideal.

**Comment by wasade** [comment]
Should this be part of conda-recipes or would it be better to contribute to
bioconda?

**Comment by colinbrislawn** [comment]
I guess this is a good question for whole conda-recipes project. Should we be making recipes here or over with the bioconda folks? Biocoda already maintains a [vsearch-recipe](https://github.com/bioconda/bioconda-recipes/tree/master/recipes/vsearch) so we know some work is being duplicated...
